engine/src/configuration/configuration.cpp

```diff
@@ -11,7 +11,7 @@ Configuration::Configuration()
     //logging.verbose_debug = GameConfig::GetVariable("data", "verbose_debug", false);
 }
 
-Configuration::CollisionHacks::CollisionHacks():
+vegastrike::configuration::CollisionHacks::CollisionHacks():
```

Comment:
You should use a `using namespace vegastrike::configuration` at the top then this doesn't have to be here. There's a few places where that doesn't work but it's extremely rare.

Not a blocker.

Reply:
Unfortunately, this is one of these rare cases. As I mentioned before, one of the structs in the namespace is Computer and we also have a Computer class outside.